Extensions/old_stats.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Old Stats **//
-//* VERSION 0.2.2 **//
+//* VERSION 0.3.0 **//
```

Comment:
pre-1.0.0 there's no concept of a "major" change besides committing to 1.0.0/stability, so a minor increment makes perfect sense here.

also a completely stupid argument, since this extension is no longer user facing, and probably will be removed from the gallery soon.